docs/user-guide/docs-source/setup.md

```diff
@@ -0,0 +1,123 @@
+# Setup
+
+
+
+## Initial Setup (One-Time)
+
+Complete the following setup before using Ambit for the first time.
+
+>  ⏱ Initial setup takes about 20 minutes.
+
+> 🛑 These instructions assume you already have Unreal Engine 4.27 installed. If you don't, you can [download Unreal Engine for free](https://www.unrealengine.com/en-US/download). Choose the Unreal Engine license that is appropriate for your team. Ambit will work with any Unreal Engine license.
+
+### 1. Download the Ambit plugin for UE4
+
+Download the latest pre-built release of the Ambit UE4 plugin from the GitHub [Releases section](https://github.com/aws-samples/aws-ambit-scenario-designer-ue4/releases).
+
+Unzip the file you downloaded. You may unzip it to any location that's convenient for you. It will unzip as a folder named "AwsAmbit".
+
+We'll return to the Ambit plugin in later steps.
+
+### 2. Install the AWS Command Line Interface
+
+You will be configuring your computer to communicate with AWS services. The easiest way to do this is by installing the AWS Command Line Interface (CLI).
+
+Install the AWS CLI to your local computer following the ["Install/Update" instructions on this page](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html). (You can ignore the "Next steps" section at the bottom of that page.)
+
+### 3. Create AWS credentials for use with Ambit
+
+Ambit integrates with Amazon's cloud storage service, [Amazon Simple Storage Service (S3)](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html). To enable this integration, you will need to provide AWS credentials that allow access to S3. 
```

Comment:
"Amazon Simple Storage Service (S3)" => "Amazon Simple Storage Service (Amazon S3)"
Name should be consistently "Amazon S3" throughout code and documentation.

Reply:
I will update the references in this doc for this PR. I'll sweep the rest of the user guide for references as part of my next PR.

Reply:
Everything else should be consistently "Amazon S3" and "S3 bucket" in our code and documentation, as I did a sweep a few weeks ago. Feel free to double-check, but do not feel pressured as part of this PR.